tutorials/CMakeLists.txt

```diff
@@ -19,12 +19,12 @@ if(DEFINED ROOT_SOURCE_DIR)  # Testing using the binary tree
   if(pyroot_experimental)
```

Comment:
Shouldn't we just remove this `if` completely? The `else` already takes care of everything.